tests/asahirov/test_homework_4.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+
+def test_collections_sort():
+    # sort data (same for min max)
+    data = [{"age": 16, "name": "John", "sex": "M"},
+            {"age": 34, "name": "Marry", "sex": "F"},
+            {"age": 25, "name": "Mathew", "sex": "M"}]
+
+    sorted_list = sorted(data, key=lambda i: i["age"])
+    assert sorted_list[-1]["age"] == 34
+
+    sorted_list_reverse = sorted(data, key=lambda i: i["age"], reverse=True)
+    assert sorted_list_reverse[-1]["age"] == 16
+
```

Comment:
assert [i["age"] for i in sorted_list] == [34, 25, 16]

Reply:
updated